Realm.Shared/Dynamic/MetaRealmObject.cs

```diff
@@ -50,64 +51,63 @@ public override DynamicMetaObject BindGetMember(GetMemberBinder binder)
             var arguments = new List<Expression>
             {
                 WeakConstant(_metadata.Table),
-                Expression.Field(GetLimitedSelf(), RealmObjectRowHandleField),
-                Expression.Constant(_metadata.ColumnIndices[property.Name])
+                Expression.Constant(_metadata.ColumnIndices[property.Name]),
```

Comment:
Is this a criticism of the code or an explanation, which should be saved in code comments rather than here?